docs/standard/async.md

```diff
@@ -30,9 +30,8 @@ Async code has the following characteritics:
 * Handles more server requests by yielding threads to handle more requests while waiting for I/O requests to return.
```

Comment:
Line 28 (earlier): characteritics --> characteristics
We're stressing how easy async code is, but it's only the task-based async pattern with compiler support that's easy. In line 26, I'd change "Async APIs and the language level..." to "Task-based async APIs and the language-level..."